### ISSUE TITLE
Enabling drag-and-drop functionality for the Cluster Explorer, Cluster Management sections would enhance convenience.

### ISSUE THREAD

**Issue by anupama2501** [rancher/dashboard#9687]
**Is your feature request related to a problem? Please describe.**
With the latest dashboard updates in version 2.8, there is some extra space allocated to the cluster management page, cluster explorer page, and the hamburger menu page.

**Describe the solution you'd like**
It would be convenient if the area could be easily moved by dragging.

**Additional context**
From the screenshots, the space allocated to the pages is too much, if its drag and drop, if there is an option provided to adjust to the required space will be more convenient.

<img width="292" alt="2023-09-12_09-17-01" src="https://github.com/rancher/dashboard/assets/38144301/0cd9c9d4-2bad-4644-ae0a-5e5255460516">
<img width="297" alt="2023-09-12_09-17-36" src="https://github.com/rancher/dashboard/assets/38144301/72d1be19-8234-4525-bbdc-82a4df3c1ac6">

**Comment by richard-cox** [comment]
I think in the 2.8.0 timeframe we should reduce the width of all non-cluster explorer side menu's.

Post 2.8.0 we should consider letting the user manually change this. We would need to persist the users change as well

**Comment by kwwii** [comment]
If I understand this correctly, you would like to resize both the app-bar (when open) and the menu?

**Comment by kwwii** [comment]
The width of the app-bar or menu should (almost) never be smaller than their content. Your screenshots do not show the longer text labels which are shown within those areas.

**Comment by kwwii** [comment]
In the next iteration, we should investigate making the sidenav menu resizeable. The app bar has a minimized state already and shouldn't need this kind of functionality.

**Comment by anupama2501** [comment]
@kwwii that is correct, I would like them - both app-bar and the menu to be resized. 

However, as previously mentioned in the comments above, as it's important to note that the space is required for both 'Virtualization Management' and the 'About' page, we can disregard the app bar section.

**Comment by nwmac** [comment]
Realistically, I don't think we will do this - suggest we close as won't fix
